### PR TITLE
create a Rails guide

### LINKED ISSUE #1: move in the Rails deployment notes
TODO for myself, mostly.

https://docs.google.com/a/gsa.gov/document/d/1J81HE-XyKan8u-DKOoV5iqn2kFqJDthwqLOLEnl6JUc/edit

### PR DESCRIPTION
**Pull request by afeld**
Closes https://github.com/18F/cloud-foundry-notes/issues/1.

Moved from [Google Docs](https://docs.google.com/a/gsa.gov/document/d/1J81HE-XyKan8u-DKOoV5iqn2kFqJDthwqLOLEnl6JUc/edit). Also, extracted a guide from the Django page on setting up databases, which is now shared between the two.

/cc @adelevie @kaitlin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,26 +23,26 @@ tar -zxvf cf.tgz
...
 ## Setting up your account
 
-You will need a Cloud Foundry account before continuing. Please create an issue in the [DevOps issue tracker](https://github.com/18F/DevOps/issues) and assign it to [@dlapiduz](https://github.com/dlapiduz) or [@ozzyjohnson](https://github.com/ozzyjohnson).
+You will need a Cloud Foundry account before continuing. Please [create an issue in the DevOps issue tracker](https://github.com/18F/DevOps/issues/new) and assign it to [@dlapiduz](https://github.com/dlapiduz) or [@ozzyjohnson](https://github.com/ozzyjohnson).
 
-To login, run:
+### Log in
 
 ```bash
 cf api --skip-ssl-validation https://api.cf.18f.us
```
Comment (adelevie) [inline README.md:41]: Is the `--skip-ssl` flag still needed?

File: app-deployment/databases.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Databases
+
+While databases can be spun up within Cloud Foundry via [services](http://docs.cloudfoundry.org/devguide/services/), the preference is to use AWS-managed resources when possible. These make things like automatic backups easy to set up, and have a more sane default configuration than the default Postgres/MySQL install on Ubuntu.
+
+## RDS
+
+1. Create the database instance on AWS.
+1. Connect the database to the `CloudFoundry-live` VPC.
```
Comment (adelevie) [inline app-deployment/databases.md]: How does one do this?

File: app-deployment/databases.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+1. Set the `DATABASE_URL` environment variable:
+
+    ```bash
+    cf set-env <APP_NAME> DATABASE_URL postgresql://<USERNAME>:<PASSWORD>@<HOST>:5432/<NAME>
```
Comment (adelevie) [inline app-deployment/databases.md]: Where do we get `<USERNAME>`, `<PASSWORD>`, and `<HOST>` from? AWS?

**Comment by adelevie** [comment]
Great work, @afeld! I left a few comments for things that weren't obvious to me. Address or ignore them at will. Thanks for writing this.

File: README.md (hunk 1 above)
Comment (afeld) [inline README.md:41]: Seems so:

```
$ cf api https://api.cf.18f.us
Setting api endpoint to https://api.cf.18f.us...
FAILED
Invalid SSL Cert for api.cf.18f.us
TIP: Use 'cf api --skip-ssl-validation' to continue with an insecure API endpoint
```

See https://trello.com/c/eFV5ze6b/16-get-ssl-cert-for-api-labs-18f-us.

Comment (konklone) [inline README.md:41]: Have we figured out what our final domain will be for the PaaS? Something at 18f.gov?

Comment (afeld) [inline README.md:41]: See https://github.com/18F/DevOps/issues/219.

**Comment by afeld** [comment]
Updated!

> ignore them at will

I shant! You're the target audience, sir.

**Comment by dlapiduz** [comment]
@afeld I wonder if we can split some of this up into more generic stuff. Like "Setting up your RDS instance" instead of being Rails specific.

**Comment by dlapiduz** [comment]
@afeld actually, I am going to merge this and do some restructuring after. We can break it apart once that is done.

**Pull request merged**